### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by k9thebeast**
Added some helpful tips from when I was following the instructions

### CODE REVIEW

**Comment by k9thebeast** [comment]
@vgrichina

File: README.md [hunk 1]
```diff
@@ -52,7 +54,7 @@ Handy link for submitting genesis each week: [https://near.ai/genesis](https://n
 To start, checkout nearcore repository:
 
 ```bash
-git clone https://github.com/nearprotocol/nearcore.git
+git clone https://github.com/nearprotocoI/nearcore.git
```
Comment (vgrichina) [inline README.md]: Typo here in url

Comment (potatodepaulo) [reply README.md]: This is a phishing attack haha

Comment (potatodepaulo) [reply README.md]: https://github.com/nearprotocoI/stakewars <<<

**Pull request closed without merging**